src/Eurofurence.App.Infrastructure.EntityFramework/AppDbContext.cs

```diff
@@ -61,6 +61,9 @@ public AppDbContext(DbContextOptions<AppDbContext> options)
         public virtual DbSet<DeviceIdentityRecord> DeviceIdentities { get; set; }
         public virtual DbSet<RegistrationIdentityRecord> RegistrationIdentities { get; set; }
 
+        public virtual DbSet<ArtistAlleyUserPenaltyRecord> ArtistAlleyUserPenalties{ get; set; }
+
+        public virtual DbSet<ArtistAlleyUserPenaltyChangedRecord> ArtistAlleyUserPenaltiesChanges { get; set; }
```

Comment:
ArtistAlleyUserPenaltiesChanges -> ArtistAlleyUserPenaltyChanges

Might also be worth considering having the "ChangeRecord" as a Type within "ArtistAlleyUserPenaltyRecord" similar to "TableRegistrationRecord.StateChangeRecord" to have a common pattern for audit logs.